### PR TITLE
Add motor test script

### PR DESCRIPTION
**Pull request by ishchow**

### CODE REVIEW

File: scripts/test_motor.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Device.pin_factory = RPiGPIOFactory()
+
+    driver = EasyDriver(step_pin=STEP_PIN,
+                dir_pin=DIR_PIN, 
+                ms1_pin=MS1_PIN, 
+                ms2_pin=MS2_PIN,
+                enable_pin=ENABLE_PIN)
+
+    print("full-step fwd")
+    driver.microstep_resolution = MicroStepResolution.FULL_STEP
```
Comment (AY-Chen) [inline scripts/test_motor.py]: How do the resolutions work? When we do like half step, do we actually have 400/steps per rotation?

File: easydriver/easydriver.py [hunk 2]
```diff
@@ -249,7 +249,7 @@ def _step_once(self):
         steps {int} -- number of steps
         direction {StepDirection} -- direction to step in
     """
-    def step(self, steps, direction=StepDirection.FORWARD):
+    def step(self, steps=1, direction=StepDirection.FORWARD):
```
Comment (AY-Chen) [inline easydriver/easydriver.py]: Is there any particular reason we're leaving the step direction as an optional parameter?

File: scripts/test_motor.py (hunk 1 above)
Comment (ishchow) [reply scripts/test_motor.py]: Basically, microstepping can half the step of your angle each step. full step = 1.8 degrees. half-step = 0.9 degress. quarter-step = 0.45 degrees.

I don't find that half-step and quarter-step work all that well though. It seems that full-step works more reliably.

File: easydriver/easydriver.py (hunk 2 above)
Comment (ishchow) [reply easydriver/easydriver.py]: Not really, it was kind of an arbitrary decision.

**Pull request merged**